src/crypto/PublicKey.ts

```diff
@@ -211,6 +217,123 @@ export class SignedPublicKey
   }
 }
 
+export class AccountLinkedPublicKeyV1
+  extends UnsignedPublicKey
+  implements publicKey.AccountLinkedPublicKey_V1
+{
+  keyBytes: Uint8Array
+  staticSignature: AccountLinkedStaticSignature | undefined
+  siweSignature: signature.AccountLinkedSIWESignature | undefined
+
+  constructor(obj: publicKey.AccountLinkedPublicKey_V1) {
+    if (!obj.keyBytes) {
+      throw new Error('Invalid AccountLinkedPublicKeyV1 has no keyBytes')
+    }
+    super(publicKey.UnsignedPublicKey.decode(obj.keyBytes))
```

Comment:
TIL super doesn't have to be the first statement in a constructor (apparently fixed by this guy [PR](https://blog.joshuakgoldberg.com/code-before-super/))